controllers/routes.js

```diff
@@ -1,358 +1,244 @@
+        path: '/empowering-young-people',
+        template: 'pages/toplevel/eyp',
+        lang: 'toplevel.eyp',
+        aliases: ['/global-content/programmes/northern-ireland/empowering-young-people']
+    }),
+    helpingWorkingFamilies: staticRoute({
+        path: '/helping-working-families',
+        template: 'pages/toplevel/working-families',
+        lang: 'toplevel.helpingWorkingFamilies',
+        aliases: ['/global-content/programmes/wales/helping-working-families']
+    })
 };
 
-// configure base paths for site sections
-const sectionPaths = {
-    toplevel: '',
-    funding: '/funding',
-    research: '/research',
-    about: '/about-big', // @TODO rename on launch
-    aboutLegacy: '/about-big' // used on the old site
+/**
+ * Funding Routes
+ */
+sections.funding.pages = {
+    root: dynamicRoute({
+        path: '/',
+        template: 'pages/toplevel/funding',
+        lang: 'toplevel.funding',
+        aliases: ['/home/funding']
+    }),
+    applyingForFunding: staticRoute({
+        path: '/funding-guidance/applying-for-funding/*'
+    }),
+    manageFunding: staticRoute({
+        path: '/funding-guidance/managing-your-funding',
+        template: 'pages/funding/guidance/managing-your-funding',
+        lang: 'funding.guidance.managing-your-funding',
+        aliases: ['/funding/funding-guidance/managing-your-funding/help-with-publicity', '/welcome', '/publicity']
+    }),
+
+    // @TODO: This is the only route that uses isWildcard??
+    freeMaterials: {
+        path: '/funding-guidance/managing-your-funding/ordering-free-materials',
+        template: 'pages/funding/guidance/order-free-materials',
+        lang: 'funding.guidance.order-free-materials',
+        live: true,
+        isPostable: true,
+        isWildcard: true,
+        aliases: [
+            '/funding/funding-guidance/managing-your-funding/ordering-free-materials/bilingual-materials-for-use-in-wales',
+            '/wales/funding/funding-guidance/managing-your-funding/ordering-free-materials',
+            '/scotland/funding/funding-guidance/managing-your-funding/ordering-free-materials',
+            '/england/funding/funding-guidance/managing-your-funding/ordering-free-materials',
+            '/northernireland/funding/funding-guidance/managing-your-funding/ordering-free-materials'
+        ]
+    },
+    helpWithPublicity: staticRoute({
+        path: '/funding-guidance/managing-your-funding/social-media',
+        template: 'pages/funding/guidance/help-with-publicity',
+        lang: 'funding.guidance.help-with-publicity'
+    }),
+    pressCoverage: staticRoute({
+        path: '/funding-guidance/managing-your-funding/press',
+        template: 'pages/funding/guidance/getting-press-coverage',
+        lang: 'funding.guidance.getting-press-coverage',
+        static: true,
+        live: true
+    }),
+    logos: staticRoute({
+        path: '/funding-guidance/managing-your-funding/grant-acknowledgement-and-logos/logodownloads',
```

Comment:
Since we're tweaking stuff – this URL is actually wrong, path-wise. Could you remove the `/logodownloads` from the end (make it a redirect here instead) and have the canonical path be `/funding-guidance/managing-your-funding/grant-acknowledgement-and-logos`?